resources/events/models/event_type.yml

```diff
@@ -0,0 +1,41 @@
+type: object
+
+required:
+  - id
+  - enabled_for_test
+  - resource
+  - object
+
+properties:
+  id:
+    oneOf:
```

Comment:
```suggestion
  id:
    description: Unique identifier, full list of possible values can be found [here](#tag/Events).
    oneOf:
```

Reply:
gonna put the descriptions in the individual files instead